apps/omg_watcher/test/exit_processor/core_test.exs

```diff
@@ -766,6 +806,330 @@ defmodule OMG.Watcher.ExitProcessor.CoreTest do
     end
   end
 
+  describe "evaluates new piggybacks" do
+    @tag fixtures: [:processor_filled, :in_flight_exits]
+    test "detects none if there is no piggybacks", %{processor_filled: processor} do
+      {:ok, alerts} =
+        %ExitProcessor.Request{blknum_now: 1000, eth_height_now: 5}
+        |> Core.invalid_exits(processor)
+
+      assert [] = alerts |> Enum.filter(&match?(%Event.InvalidPiggyback{}, &1))
```

Comment:
note the `invalid_exits_filtered` (or similar name) helper that has been added, please use that. Possibly rebase is required to get that

Reply:
done